projects/bepro/index.js

```diff
@@ -0,0 +1,71 @@
+const sdk = require('@defillama/sdk');
+
+/** BEPRO Protocol is available on Moonbeam and Moonriver */
+const config = {
+    "ethereum": {
+        token: "0xcf3c8be2e2c42331da80ef210e9b1b307c03d36a",
+        bountyNetworks: []
+    },
+    "moonriver": {
+        token: "0xCb4a593ce512D78162C58384f0b2Fd6e802c2c47",
+        bountyNetworks: ["0x85dE589aDc4bC5F17075fcd603E8A0f7561d90C9"]
+    },
+    "moonbeam": {
+        token: "0x4EdF8E0778967012D46968ceadb75436d0426f88",
+        registry: "0x34DD5F63437FdC20557a8C6dDAeA056d3661c5e0",
```

Comment:
@hvasconcelos thanks for the PR, what is this registry and why are we counting the tokens in it.

Also noticed that, you are live only in the moonbeam network?

Reply:
The Registry is another contract where users lock tokens to access our services. 
Our product is live on Moonriver(v1) and Moonbeam (v2) but the token exists on Ethereum, Polygon, Aurora, Moonbeam, and Moonriver.